src/main.js

```diff
@@ -133,7 +134,10 @@ takeMeBackButtonFromMake.addEventListener('click', goBackToMainMake);
 backToMainButtonFromSaved.addEventListener('click', goBackToMainSaved);
 makeOwnPosterButton.addEventListener('click', showMakeForm);
 //Iteration 2
-showMyPosterButton.addEventListener('click', showUserPoster);
+showMyPosterButton.addEventListener('click', function(){
```

Comment:
cool. syntaxing two functions in this event clicker's argument :)

Reply:
yeah! Pretty sure it was you who asked a question about that in class yesterday, right?